gym-gazebo-server.py

```diff
@@ -82,7 +82,7 @@ def image_callback(self, msg):
 
     if msg['command'] == 'reset':
         print('resetting the simulation')
-        reset_proxy()
+        #reset_proxy()
```

Comment:
Wait, what? If you comment out the actual reset function, then yeah of course the resetting is stable. 
We have to make it stable after calling this function. :)